docs/docs/scanner/misconfiguration/index.md

```diff
@@ -343,6 +343,59 @@ You can load checks bundle as OCI Image from a Container Registry using the `--c
 trivy conf --checks-bundle-repository myregistry.local/mychecks --namespaces user myapp
 ```
 
+
+### Scan arbitrary JSON and YAML configurations
+By default, scanning JSON and YAML configurations is disabled, since Trivy does not contain built-in checks for these configurations. To enable it, pass the `json` or `yaml` to `--misconfig-scanners`. See [Enabling a subset of misconfiguration scanners](#enabling-a-subset-of-misconfiguration-scanners) for more information. Trivy will pass each file as is to the checks input.
+
+
+!!! example
+```bash
+$ cat iac/serverless.yaml
+service: serverless-rest-api-with-pynamodb
+
+frameworkVersion: ">=2.24.0"
+
+plugins:
+  - serverless-python-requirements
+...
+
+$ cat serverless.rego
+# METADATA
+# title: Serverless Framework service name not starting with "aws-"
+# description: Ensure that Serverless Framework service names start with "aws-"
+# schemas:
+#   - input: schema["serverless-schema"]
```

Comment:
We should also explicitly describe the directory structure here so that the user knows how to create and pass the custom schema, if they don't provide on via the new flag. The context here is how Rego picks up schemas based on directory structures, if you recall our previous conversation.

Reply:
The `--config-file-schemas` flag has been added for explicit passing of schemas.

Reply:
We automatically detect possible schemes for Rego by annotation in Rego checker. I don't think this method is suitable for finding a schema for a file checker.

Reply:
What I had meant to ask is we should mention the directory structure as part of this PR in its documentation. We actually have that already documented [here](https://aquasecurity.github.io/trivy/v0.54/docs/scanner/misconfiguration/custom/schema/#custom-checks-with-custom-schemas) so we could simply link to it to remind the user about it.

Reply:
I got it. I'll update the documentation.

Reply:
Added this here https://github.com/aquasecurity/trivy/pull/7311/commits/19b999902bf6012fb38952a9fc7ee2f82eed9b12